dominion/dominion_ui/src/components/Hero/CountryHero.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+
+import { PropTypes } from 'prop-types';
+import { Grid } from '@material-ui/core';
+import { withStyles } from '@material-ui/core/styles';
+
+import Hero, {
+  HeroTitle,
+  HeroDescription,
+  HeroTitleGrid,
+  HeroButton
+} from './Hero';
+
+const styles = theme => ({
+  root: {
+    flexGrow: 1
+  },
+  map: {
+    backgroundColor: 'grey',
+    height: '460px !important',
+    width: '829px !important',
+    maxHeight: '460px !important',
+    maxWidth: '829px !important',
+    left: 'unset !important',
+    top: 'unset !important',
+    [theme.breakpoints.up('sm')]: {
+      position: 'absolute',
+      right: '150px'
+    },
+    [theme.breakpoints.down('sm')]: {
+      position: 'relative !important',
+      width: '100% !important',
+      height: '250px !important'
+    }
```

Comment:
The above two `[theme.breakpoints.up('sm')]` and `[theme.breakpoints.down('sm')]` clashes for `sm` i.e. break points work from that point up or down _inclusively_.